iron-overlay-manager.html

```diff
@@ -22,6 +23,8 @@
 
     _backdrops: [],
 
+    _backdropElement: document.createElement('iron-overlay-backdrop'),
```

Comment:
the reason why this is failing is because the manager is a singleton, an object, and here we add a key. To access to it we should do `Polymer.IronOverlayManager._backdropElement` in the getter (ugly).
Much better to lazy load by checking if `this._backdropInstance` exists.
```javascript
   get backdropElement() {
      if (!this._backdropElement) {
        this._backdropElement = document.createElement('iron-overlay-backdrop');
      }
      return this._backdropElement;
    },